app/graphql/mutations/payment_providers/adyen.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Mutations
+  module PaymentProviders
+    class Adyen < BaseMutation
+      include AuthenticableApiUser
+      include RequiredOrganization
+
+      graphql_name 'AddAdyenPaymentProvider'
+      description 'Add or update Adyen payment provider'
+
+      argument :api_key, String, required: true
+      argument :merchant_account, String, required: true
+      argument :live_prefix, String, required: false
```

Comment:
GraphQL/OrderedArguments: Arguments should be sorted in an alphabetical order within their section. Field `live_prefix` should appear before `merchant_account`.